test/e2e/demo-storage-plugin/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM registry.access.redhat.com/ubi8-minimal
+COPY demo-storage-plugin /
+CMD ["cp", "/demo-storage-plugin", "/plugin/demo-storage-plugin"]
```

Comment:
I think this deserves some comments:

1. "demo-storage-plugin" is your plugin binary
1. the operator will mount an empty dir at /plugin, so, you need to copy the binary anywhere in there
1. the final path should match what you specified in the `.Spec.Storage.Binary`

Alternatively (and better), open an issue against the docs repo with this information, and add the link as a doc here.